assets/strings/eng.json

```diff
@@ -61,5 +61,7 @@
     "Apertium_Downloads": "Apertium Downloads",
     "Downloads_Para": "Current versions of the <b>Apertium toolbox</b> as well as of <b>language-pair data</b> are available from <a href='https://sourceforge.net/projects/apertium/' target='_blank'>the SourceForge page</a>. Installation instructions for Apertium on all major platforms are provided on the <a href='http://wiki.apertium.org/wiki/Installation' target='_blank'>Wiki's Installation page</a>.",
-    "Contact_Para": "<h4>IRC channel</h4> <p>The quickest way to contact us is by joining our <a href='http://wiki.apertium.org/wiki/IRC' target='_blank'><abbr title='Internet Relay Chat'>IRC</abbr> channel, #apertium</a> at irc.freenode.net, where users and developers of Apertium meet. You don't need an IRC client; you can use <a href='http://webchat.freenode.net/?channels=apertium'>freenode webchat</a>. </p> <h4>Mailing list</h4> <p>Also, subscribe to the <a href='https://lists.sourceforge.net/lists/listinfo/apertium-stuff' target='_blank'> apertium-stuff mailing list</a>, where you can post longer proposals or issues, as well as follow general Apertium discussions. </p> <h4>Contact</h4> <p>Feel free to contact us via the <a href='mailto:apertium-contact@lists.sourceforge.net' target='_blank'>apertium-contact mailing list</a> if you find a mistake, there's a project you would like to see us work on, or you would like to help out.</p>"
+    "Contact_Para": "<h4>IRC channel</h4> <p>The quickest way to contact us is by joining our <a href='http://wiki.apertium.org/wiki/IRC' target='_blank'><abbr title='Internet Relay Chat'>IRC</abbr> channel, #apertium</a> at irc.freenode.net, where users and developers of Apertium meet. You don't need an IRC client; you can use <a href='http://webchat.freenode.net/?channels=apertium'>freenode webchat</a>. </p> <h4>Mailing list</h4> <p>Also, subscribe to the <a href='https://lists.sourceforge.net/lists/listinfo/apertium-stuff' target='_blank'> apertium-stuff mailing list</a>, where you can post longer proposals or issues, as well as follow general Apertium discussions. </p> <h4>Contact</h4> <p>Feel free to contact us via the <a href='mailto:apertium-contact@lists.sourceforge.net' target='_blank'>apertium-contact mailing list</a> if you find a mistake, there's a project you would like to see us work on, or you would like to help out.</p>",
+    "Installation_Notice_Title": "Inconvenience",
+    "Installation_Notice": "We are highly sorry for the inconvenience caused but our server appears loaded now. In order to set apertium locally, please refer to this <a href='http://wiki.apertium.org/wiki/Apertium-apy' target='_blank'> link</a>."
```

Comment:
`"Are you experiencing slow responses? Our servers might be overloaded. Learn how to <a href="http://wiki.apertium.org/wiki/Installation">install Apertium locally</a>."`